### PR TITLE
Add Farfetch icon to the packages

### PR DESCRIPTION
**Pull request by martinhonovais**
# Description

Add Farfetch icon to the packages

## How Has This Been Tested?

No tests.

## Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have added tests to cover my changes
-   [ ] I have made corresponding changes to the documentation

### Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

**Comment by gsferreira** [comment]
@martinhonovais instead of replicating the change across multiple Projects, maybe adding a `Directory.Build.props` file would be a good thing.
It's just a suggestion. Not a blocker.

**Review by sergioamribeiro — APPROVED** [review]

**Review by sergioamribeiro — APPROVED** [review]

File: src/KafkaFlow.Retry.sln [hunk 1]
```diff
@@ -32,6 +32,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{9557F908-472F-4872-BCF8-8EC028EFDA9B}"
 	ProjectSection(SolutionItems) = preProject
 		CodeCoverage.runsettings = CodeCoverage.runsettings
+		Directory.Build.props = Directory.Build.props
```
Comment (gsferreira) [inline src/KafkaFlow.Retry.sln:35]: Are you sure you need this?

File: src/KafkaFlow.Retry/Durable/Definitions/Builders/RetryDurableEmbeddedClusterDefinitionBuilder.cs [hunk 2]
```diff
@@ -127,19 +127,12 @@ public RetryDurableEmbeddedClusterDefinitionBuilder WithRetryTypedHandlers(Actio
                         .WithPartitionsAssignedHandler(
                             (resolver, partitionsAssignedHandler) =>
                             {
-                                var log = resolver.Resolve<ILogHandler>();
```
Comment (gsferreira) [inline src/KafkaFlow.Retry/Durable/Definitions/Builders/RetryDurableEmbeddedClusterDefinitionBuilder.cs:130]: Just to confirm: is this change expected?

Comment (martinhonovais) [reply src/KafkaFlow.Retry/Durable/Definitions/Builders/RetryDurableEmbeddedClusterDefinitionBuilder.cs:130]: Yes, KafkaFlow is logging something on assign and revoking the partitions. It's not needed.

Comment (gsferreira) [reply src/KafkaFlow.Retry/Durable/Definitions/Builders/RetryDurableEmbeddedClusterDefinitionBuilder.cs:130]: 👍

**Review by gsferreira — APPROVED** [review]

**Review by luispfgarces — APPROVED** [review]

**Review by jdromeiro — APPROVED** [review]

**Pull request merged**